python/TestHarness/testers/Exodiff.py

```diff
@@ -51,7 +51,6 @@ def processResults(self, moose_dir, retcode, options, output):
         output = RunApp.processResults(self, moose_dir, retcode, options, output)
 
         if self.getStatus() == self.bucket_fail or self.specs['skip_checks']:
-            self.setStatus('skip checked', self.bucket_skip)
             return output
```

Comment:
Accidentally treated this as a 'skipped' test, when in fact this needs to be a failure. The status of this test would have already been set within RunApp's processResults method.